### PR TITLE
Show that Command can be reused and remodified

### PR DESCRIPTION
**Pull request by Havvy**
The prior documentation did not make it clear this was possible.

I wanted to make the `list_dir` example work on Windows, but I don't know if passing "/" will error or show the root of the current volume (e.g. `C:`).

r? @GuillaumeGomez

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
That's pretty cool, thanks!

@bors: r+ rollup

**Comment by GuillaumeGomez** [comment]
(please ping me or someone else with r+ rights once CI passed)

@bors: r-

File: src/libstd/process.rs [hunk 1]
```diff
@@ -381,6 +381,39 @@ impl fmt::Debug for ChildStderr {
...
+///
+/// `Command` can be reused to spawn multiple processes. The builder methods
+/// change the command without needing to immediately spawn the process.
+///
+/// ```no_run
+/// use std::process::Command;
+///
+/// let mut echo_hello = Command::new("sh");
+/// echo_hello.arg("-c")
+///            .arg("echo hello");
```
Comment (GuillaumeGomez) [inline src/libstd/process.rs]: Please fix indent (one extra whitespace).

**Comment by Havvy** [comment]
@GuillaumeGomez Awaiting your r+ now.

**Comment by GuillaumeGomez** [comment]
Thanks!

@bors: r+ rollup

**Pull request merged**